summary_workflows/differential_usage/specification/D_benchmark_specification.md

```diff
@@ -0,0 +1,133 @@
+
+
+## Outputs
+
+Calculated metrics are saved in JSON file adhering to OpenEBench schema. 
+Assessment output is generated for each tool separately and contains values of calculated metrics for a given input dataset.
+Consolidation output contains summarized data from all benchmarked tools within one challenge in a format suitable for plotting, e.g. single values for barplot or X,Y value pairs for 2D scatter plot.
+
+| # | Format | Link | Example data | Description |
+| :-- | :--- | :--- | :--- | :-------- |
+| 1 | JSON | [Specification][spec-json] | [Link][assessment_out] | Assessment output JSON |
+| 2 | JSON | [Specification][spec-json] | [Link][aggregation_out] | Aggregation output JSON
+| 3 | JSON | [Specification][spec-json] | [Link][consolidation_out] | Consolidation output JSON |
+
+### Additional info
+ 
+#### Output 1
+
+The OpenEBench assessment file contains the following attributes:
+
+- **\_id** - follows the format: community:challenge\_metric\_tool
+- **challenge_id** - describing the combination of input dataset with indication whether it is based on real or simulated data and any parameters used for metric calculation; e.g. datasetA\_simulated\_10nt
+- **participant_id** - benchmarked tool
+- **metrics**
+	- **value** - metric value
+	- **metric_id** - metric name; metric names used in this benchmark are specified in the table below
+ 
+The following tables list the metric names, value types and units, and a description:
+
+| Metric_id | Type | Unit | Description |
+| :--- | :--- | :--- | :----------------- |
+| `Sensitivity` | `float` | N/A | Sensitivity of detection of genes with differential PAS usage compared with orthogonal dataset; Sensitivity = (TP/(TP+FN)) |
+| `FPR` | `float` | N/A | False Positive Rate of detection of genes with differential PAS usage compared with orthogonal dataset; FPR = FP/(FP+TN) |
+ 
+#### Output 2
+
+The OpenEBench aggregation file contains information required to produce one plot.
+
+#### Output 3
+
+The OpenEBench consolidation file contains all the information about the new benchmarking run and specifies visualization types (2D scatter plot, barplot), descriptions of metrics used for X and Y axis and (X,Y) value pairs for each challenge and challenge participants.
+ 
+
+[//]: # (References)
+  
+[in1]: ./example_files/input1.tsv
+[in2]: ./example_files/input2.tsv
+[assessment_out]: ./example_files/assessment_out.json
+[aggregation_out]: ./example_files/aggregation_out.json
+[consolidation_out]: ./example_files/consolidation_out.json
```

Comment:
This file doesn't exist

Reply:
I removed the consolidation file example.